src/invidious/config.cr

```diff
@@ -56,11 +56,13 @@ end
 class Config
   include YAML::Serializable
 
-  property channel_threads : Int32 = 1           # Number of threads to use for crawling videos from channels (for updating subscriptions)
-  property feed_threads : Int32 = 1              # Number of threads to use for updating feeds
-  property output : String = "STDOUT"            # Log file path or STDOUT
-  property log_level : LogLevel = LogLevel::Info # Default log level, valid YAML values are ints and strings, see src/invidious/helpers/logger.cr
-  property db : DBConfig? = nil                  # Database configuration with separate parameters (username, hostname, etc)
+  property channel_threads : Int32 = 1 # Number of threads to use for crawling videos from channels (for updating subscriptions)
+  @[YAML::Field(converter: TimeSpanConverter)]
+  property channel_refresh_time : Time::Span = 30.minutes # Time between two jobs for crawling videos from channels
```

Comment:
While you're at it, could you please put the comments above their respective variable, rather than inline? (note that it has to be before the `@[]` converter statement).

Reply:
No problem, I was following the way it was done at that place